src/plugins/importFromJSONAction/ImportFromJSONAction.js

```diff
@@ -43,7 +43,12 @@ export default class ImportAsJSONAction {
      */
     appliesTo(objectPath) {
         const domainObject = objectPath[0];
-        if (domainObject && domainObject.locked) {
+        const locked = domainObject && domainObject.locked;
+        const persistable = this.openmct.objects.isPersistable(domainObject.identifier);
+        const TypeDefinition = this.openmct.types.get(domainObject.type).definition;
```

Comment:
handle 'this.openmct.types.get(domainObject.type)' is null?

Reply:
added the mock like you suggested, I don't think it should ver come back null, very odd.